multi-cloud/README.md

```diff
@@ -452,7 +496,7 @@ A non-zero percentage of you probably came here looking for Tensorflow on Kubern
 Now we train a Tensorflow model really quickly:
 
 ```bash
-argo submit training-tf-mnist-workflow.yaml -n kubeflow -p build-push-image=true
```

Comment:
no

Reply:
wait, yeah we don't want to build the image here right?

Reply:
thoughts here?

Reply:
I say go without building the image for now, we can fix it later if we need to.